scripts/s3mi

```diff
@@ -353,10 +354,15 @@ def get_credentials():
     [Acknowledgement:  Thanks Andrey Kislyuk, David Rissato Cruz]
     """
     if not have_boto:
-        # This effectively disables credentials caching/update.
-        return {"vars": {}, "expiration_time": time.time() + 365*24*3600}
+        tsprint("Refreshing credentials is impossible without boto.  Disabling.")
+        return DISABLED
     session = botocore.session.Session()
     credentials = session.get_credentials()
+    if not credentials or not credentials.token:
```

Comment:
The specific incident which prompted this was, credentials existed, but credentials.token was None.